tests/database/04-lock-recover/suite.rc

```diff
@@ -12,6 +12,7 @@ locker:lock => TRUES:succeed-all => done
     [[locker]]
         script = """
 timeout 60 cylc-db-lock
+cylc message "$CYLC_TASK_CYCLE_POINT I have locked the public database file"
```

Comment:
`cylc-db-lock` calls `cylc task message "[1] I have locked the public database file"` in Python. Have I done this wrong when I wrote this test? Ditto for the test 05.